src/app.js

```diff
@@ -1 +1,44 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios"
+//Stap 1. Installeer en importeer Axios;
+console.log('Hallo daar!');
+
+// Stap 3. Schrijf een asynchrone functie die, met behulp van Axios, een GET-request maakt naar het juiste endpoint. Log de response in de console en bestudeer de data goed.
+async function getCountries(){
+    const countryList = document.getElementById("country-list");
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all');
+        const countries = result.data;
+        // console.log(result.data);
+
+        countries.sort((a, b)=>{
+            return a.population - b.population;
+        });
+        result.data.map((country) => {
+            const countryItem = document.createElement("li");
+            countryItem.innerHTML = `
+            <img src="${country.flag}" alt="vlaggetje van landje" class="flag"/>
+            <p class="${regionName(country.region)}">${country.name}</p>
+            <p>Has a population of ${country.population} people</p>
+            `;
+            countryList.appendChild(countryItem)
+        })
+    } catch(e) {
+        console.error(e);
+    }
+}
+
+getCountries();
+function regionName(arrRegion) {
+
+    if (arrRegion === "Africa") {
+        return (arrRegion === "Africa");
+    } else if (arrRegion === "Asia") {
+        return "Asia";
+    } else if (arrRegion === "Europe") {
+        return "Europe";
+    } else if (arrRegion === "Oceania") {
+        return  "Oceania";
+    } else {
+        return "Americas";
+    }
```

Comment:
Het lijkt erop dat je altijd gewoon de binnengekomen waarde terug wil geven. Als je `Americas` terug wilt krijgen als het niet één van de bovenstaande regio's is, dan zou ik dit doen:
```
const regions = ["Africa", "Asia", "Europe", "Oceania"];

return regions.includes(arrRegion) ? arrRegion : "Americas";
```